### ISSUE TITLE
Provide automated environment for building protoc binaries.

### ISSUE THREAD

**Issue by kjopek** [os72/protoc-jar#38]
I believe we should prepare virtualized environment for building binaries of protoc for different architectures. At beginning - x86_64 should be enough.

I can assist you with FreeBSD builder.

**Comment by os72** [comment]
You mean VM images for virtualbox or something like that?

**Comment by kjopek** [comment]
No, just running VM that allows you to get compilation of protobuf for FreeBSD.

**Comment by os72** [comment]
Are you offering to host and run the FreeBSD VM?

**Comment by kjopek** [comment]
I have VM running FreeBSD, just need some time to prepare small app that takes tag and returns stripped protoc. I guess this would be enough. What do you think?

**Comment by os72** [comment]
Sounds great! protoc build cloud :)
